docs/building.rst

```diff
@@ -481,6 +481,19 @@ If you enabled test in the configuration step, to run them, run::
 
 	ctest -j8
 
+building with VCPKG
```

Comment:
this should probably be a heading, and the same level as the headings using `-`. Could you add a line of 19 `-` under this?

Reply:
@arvidn, thank you for your review!

I have updated the format.